source/routing/raptor.h

```diff
@@ -70,7 +70,8 @@ struct RAPTOR
 {
     const navitia::type::Data& data;
 
-    CachedNextStopTime cached_next_st;
+    CachedNextStopTimeManager cached_next_st_manager;
+    const CachedNextStopTime* current_cache;
```

Comment:
you should init it with nullptr

I prefer the previous name for this (`next_st`), I think the cache thing only concern the CachedNextStopTimeManager, not the thing cached